.gitignore

```diff
@@ -71,6 +71,7 @@ id_rsa_*
 
 # python things
 __pycache__
+.python-version
```

Comment:
Happy to remove this change if you guys feel strongly about it.

Reply:
If you want to get the new board revision support in as quick as possible, I think that removing the `.gitignore` change from this PR would be the right thing.

Reply:
And yeah, this flags it as core changes and requires targeting develop.